Gruntfile.js

```diff
@@ -39,144 +57,181 @@ module.exports = function( grunt ) {
           "unqualified-attributes": false,
           "vendor-prefix": false,
           "zero-units": false
-        },
-        src: [
-          "public/learning_projects/**/*.css"
-        ]
+        }
       }
     },
-    lesslint: {
-      src: ["public/stylesheets/userbar.less"],
+    jshint: {
+      server: {
         options: {
-          csslint: {
-            "adjoining-classes": false,
-            "box-model": false,
-            "box-sizing": false,
-            "bulletproof-font-face": false,
-            "compatible-vendor-prefixes": false,
-            "floats": false,
-            "font-sizes": false,
-            "ids": false,
-            "important": false,
-            "outline-none": false,
-            "overqualified-elements": false,
-            "qualified-headings": false,
-            "regex-selectors": false,
-            "star-property-hack": false,
-            "underscore-property-hack": false,
-            "universal-selector": false,
-            "unique-headings": false,
-            "unqualified-attributes": false,
-            "vendor-prefix": false,
-            "zero-units": false
-          }
+          "-W069": true, // ignore "['...'] is better written in dot notation." warnings
+          // Enforcing
+          bitwise       : false,     // true: Prohibit bitwise operators (&, |, ^, etc.)
+          camelcase     : false,    // true: Identifiers must be in camelCase
+          curly         : true,     // true: Require {} for every new block or scope
+          eqeqeq        : false,     // true: Require triple equals (===) for comparison
+          forin         : true,     // true: Require filtering for..in loops with obj.hasOwnProperty()
+          freeze        : false,     // true: prohibits overwriting prototypes of native objects such as Array, Date etc.
+          immed         : false,    // true: Require immediate invocations to be wrapped in parens e.g. `(function () { } ());`
+          indent        : 2,        // {int} Number of spaces to use for indentation
+          latedef       : false,    // true: Require variables/functions to be defined before being used
+          newcap        : true,    // true: Require capitalization of all constructor functions e.g. `new F()`
+          noarg         : true,     // true: Prohibit use of `arguments.caller` and `arguments.callee`
+          noempty       : false,     // true: Prohibit use of empty blocks
+          nonbsp        : false,     // true: Prohibit non-breaking whitespace characters.
+          nonew         : false,    // true: Prohibit use of constructors for side-effects (without assignment)
+          plusplus      : false,    // true: Prohibit use of `++` & `--`
+          quotmark      : false,    // Quotation mark consistency:
```

Comment:
No in that case we can just escape it inside the string! I dunno, I found it really awkward when adding code to the different files with different quote conventions and then finding some files actually having both quotes for no apparent reason.